mantidproject/mantid
# Indirect Corrections ApplyPaalmanPings -  Container plotting and Shift

The ApplyPaalmanPings interface should now allow for the previewing of spectra if shift is checked but the algorithm has not yet been run. Furthermore the container should now be loaded and plotted in the miniplot when the file is loaded by the interface

**To test:**

**All data is available from `\\olympic\babylon5\Public\ElliotOram\Corrections\CalculatePaalmanPings\IRIS\correction_set`**
- Open ApplyPaalmanPings (Interface > Indirect > Corrections > ApplyPaalmanPings)
- Load in the Sample: `IRS26176_graphite002_red.nxs`
- Check Use Container and Load `IRS26174_graphite002_red.nxs`
- Ensure the sample and container are now loaded into the miniplot.
- Check the Use scale and the use shift option and change their values slightly.
- check you can still change the Spectrum in the miniplot using the spinner in the bottom left of the interface.
- Run the interface.
- Ensure it runs without error and the 3 curves (black, red and green) are all in the mini plot
- Try various combinations of running, changing values and switching spectrum preview to ensure nothing crashes and the interface functions as expected.
- The corrections file (`IRS26176_graphite002_flt_abs.nxs`) is in the same directory and this can also be used to test the interface further.

Fixes #16255

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/8387410b2e307a60d2ce4d709e39bf06b903233d/docs/source/release/v3.7.0/indirect_inelastic.rst#bugfixes)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
